doc/libdnf5_plugins/actions.8.rst

```diff
@@ -0,0 +1,163 @@
+..
+    Copyright Contributors to the libdnf project.
+
+    This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+    Libdnf is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    Libdnf is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+.. _actions_plugin_ref-label:
+
+======================
+libdnf5 actions Plugin
+======================
+
+Description
+===========
+
+The plugin allows to define actions to be executed during libdnf5 plugin callbacks.
+Each action is hooked to one specific callback. Actions for pre_transaction and
+post_transaction callbacks may define a (glob-like) filtering rule on the package
+NEVRA or package files, as well as whether the package is incomming or outgoing.
+
+
+Configuration
+=============
+
+The plugin does not extend the standard configuration. However, it reads "actions" files.
+
+The action files are readed from ``<libdnf5_plugins_config_dir>/actions.d/`` directory. Only files
+with ".actions" extension are readed.
```

Comment:
"are readed" -> "are read" (used twice)

Reply:
Also, you first use `"actions" files`, but then it's always referred to as `action files`. It would be better to be consistent.

Reply:
The second occurrence of "readed" remained: Only files with ".actions" extension are **readed**.